app/scripts/metamask-controller.js

```diff
@@ -2620,9 +2673,12 @@ export default class MetamaskController extends EventEmitter {
    * @param {string} origin - The origin identifier for which network state is requested (default: 'metamask').
    * @returns {object} An object containing important network state properties, including chainId and networkVersion.
    */
-  getProviderNetworkState(origin = 'metamask') {
+  getProviderNetworkState(origin = METAMASK_DOMAIN) {
     let chainId;
-    if (this.preferencesController.getUseRequestQueue()) {
+    if (
+      this.preferencesController?.getUseRequestQueue() &&
```

Comment:
```suggestion
      this.preferencesController.getUseRequestQueue() &&
```
`this.preferencesController` should always exist

Reply:
unless, as appears to be the case here in some scenarios, this method is called before the controller is instantiated.

Reply:
not seeing this issue anymore so removing this for now: https://github.com/MetaMask/metamask-extension/pull/23130/commits/f2c2dbdb049f2e7268be45d684c8714acc1833f9